packages/mongo/mongo.d.ts

```diff
@@ -32,6 +32,7 @@ export namespace Mongo {
     limit?: number | undefined;
     /** Dictionary of fields to return or exclude. */
     fields?: FieldSpecifier | undefined;
+    projection?: FieldSpecifier | undefined;
```

Comment:
It'd be great to:
1. Add JSDoc to `projection` just like for the `fields`.
2. Add `@deprecated` to `fields` and point it to `projection`.

Reply:
If it is just here then it is done.